### ISSUE TITLE
404 Page Not Found in Preview

### ISSUE THREAD

**Issue by dhoffmann1979** [in2code-de/luxletter#169]
In a fresh TYPO3 v12 instance i receive the following error in the preview are when coming to step 3:

404
Page Not Found
The page did not exist or was inaccessible. Reason: Request parameters could not be validated (&cHash empty)

When [FE][pageNotFoundOnCHashError] is set to false the preview is rendered correctly. I tried the actual master & develop branch.

**Comment by einpraegsam** [comment]
I just tested in my TYPO3 12 instance but I couldn't reproduce this error.
I tested with and without cache. Any hints how to reproduce this problem?

![Screenshot 2023-02-12 at 12-38-12 Install Tool on site luxcore localhost de · luxcore localhost de TYPO3 CMS 12 2 0](https://user-images.githubusercontent.com/89055/218308670-4c9c871a-848f-4c23-8d32-ea234808632f.png)
![Screenshot 2023-02-12 at 12-18-46 luxcore localhost de TYPO3 CMS 12 2 0](https://user-images.githubusercontent.com/89055/218308674-002d212a-0fb9-4fd3-8c2b-aca7ef487331.png)
